packages/provider/src/tasks/tasks.ts

```diff
@@ -74,6 +75,8 @@ export class Tasks {
 
     config: ProsopoConfigOutput
 
+    keyring: Keyring
```

Comment:
Why are you adding keyring to Tasks. I can't see where it's used.

Reply:
Fixed